test/blackbox-tests/test-cases/pkg/run-local.t

```diff
@@ -0,0 +1,23 @@
+Test that local commands, such as `./configure`, are called from the sandbox, as
+they could have been created or modified by previous build commands (such as
+`patch`)
+
+  $ . ./helpers.sh
+
+  $ make_lockdir
+  $ cat >dune.lock/test.pkg <<EOF
+  > (version 0.0.1)
+  > (build
+  >  (progn
+  >   (write-file configure "#!/bin/sh\necho Package configured\n")
+  >   (run chmod +x configure)
+  >   (run sh configure)
+  >   (run ./configure)))
```

Comment:
Why is it running `configure` twice here? Is one failing or are both failing?

Reply:
It was my work around. You’re right: that was cryptic. I made all that explicit.

Reply:
Ah, this makes a lot more sense and is well understandable now. Thanks!